static/js/script.js

```diff
@@ -20,3 +20,15 @@ socialIcon.addEventListener('click', (event) => {
     event.preventDefault();
     this.document.querySelector('label').classList.toggle('active');
 });
+
```

Comment:
Here I would like to do this but in vanilla js (pure js) : 
```$("input").focus(function(){
  $(this).parent().addClass("active completed");
});

$("input").focusout(function(){
  if($(this).val() === "")
    $(this).parent().removeClass("completed");
  $(this).parent().removeClass("active");
})```
Any idea on how I could have this working ?

Reply:
If it's only one input element:
```js
const input = document.querySelector('input')
input.addEventListener('focus', event => {
  event.target.parent.classList.add('active', 'complete')
})

input.addEventListener('blur' => {
  if(event.target.value === '') {
    event.target.classList.remove('completed', 'active')
})
```

Reply:
you don't need any semicolon at the end of every line then ? and for `'blur' => {` don't you want need : `'blur', (event) => {` ?

Reply:
i used : https://codepen.io/legeoffrey/pen/VYMLNq?q=input&limit=all&type=type-pens as an example. And I want to use it for cfbb login form

Reply:
Here's a Vanilla equivalent: https://codepen.io/anon/pen/xPyVVY/